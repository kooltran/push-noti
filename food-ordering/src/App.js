import React, { useEffect } from 'react'
import MenuList from './components/menu/menu'
import './App.css'

const App = () => {
  return (
    <div>
      <h1>Food Ordering</h1>
      <MenuList />
    </div>
  )
}

export default App
