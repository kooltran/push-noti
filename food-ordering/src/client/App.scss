@import url('https://fonts.googleapis.com/css2?family=Comic+Neue:ital,wght@0,400;0,700;1,400&display=swap');

body {
  font-family: 'Comic Neue', cursive;
}

.page {
  margin: 50px 0 20px;
}

.Toastify__toast {
  border-radius: 5px;
  font-size: 20px;
  padding: 20px;
}
