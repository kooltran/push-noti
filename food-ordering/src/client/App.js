import React from 'react'
import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
import Home from './pages/Home/Home'
import Orders from './pages/Orders/Orders'
import OrderList from './pages/Orders/OrderList'

import PrivateRoute from './components/PrivateRoute/PrivateRoute'

import './App.scss'
import 'bootstrap/dist/css/bootstrap.min.css'

const App = () => {
  return (
    <div className="container">
      <Router>
        <Switch>
          <Route exact path="/">
            <Home />
          </Route>
          <Route path="/orders">
            <Orders />
          </Route>
          <PrivateRoute path="/allOrders" component={OrderList} />
        </Switch>
      </Router>
    </div>
  )
}

export default App
