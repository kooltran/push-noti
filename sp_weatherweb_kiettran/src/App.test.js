import React from "react";
import renderer from "react-test-renderer";
import App from "./App";

it("should render App correctly", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
