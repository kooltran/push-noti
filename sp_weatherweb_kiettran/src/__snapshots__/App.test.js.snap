// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render App correctly 1`] = `
<div
  className="layout-container"
>
  <div
    className="layout-uppper"
  >
    <div
      className="nav-container"
    >
      <div
        className="nav-burger"
      >
        <span />
        <span />
        <span />
      </div>
      <div
        className="nav-location"
      >
        <div
          className="nav-location-name"
        >
          myENV
        </div>
        <div
          className="nav-location-dropdown"
        >
          <span>
            Current Location
          </span>
          <span
            className="icon-arrow"
          />
        </div>
      </div>
      <div
        className="nav-notification"
      />
    </div>
    <div
      className="weatherStatus-wrapper"
    >
      <div
        className="weatherStatus-container"
      >
        <div
          className="weatherStatus-icon"
        />
        <div
          className="weatherStatus-info"
        >
          <div
            className="status-text"
          >
            Cloudy
          </div>
          <div
            className="status-info"
          >
            <span
              className="status-temperature"
            >
              29.9 C
            </span>
            <span
              className="status-humidity"
            >
              73%
            </span>
          </div>
        </div>
      </div>
    </div>
    <div
      className="weatherParams-wrapper"
    >
      <div
        className="weatherParams-item"
      >
        <div
          className="title"
        >
          PSI
        </div>
        <div
          className="content bg-numb"
        >
          23
        </div>
        <div
          className="extra"
        >
          Good
        </div>
      </div>
      <div
        className="weatherParams-item"
      >
        <div
          className="title"
        >
          RAIN
        </div>
        <div
          className="content normal-numb"
        >
          0
        </div>
        <div
          className="extra"
        >
          mm
        </div>
      </div>
      <div
        className="weatherParams-item"
      >
        <div
          className="title"
        >
          DENGUE
        </div>
        <div
          className="content circle-shape"
        />
        <div
          className="extra"
        />
      </div>
      <div
        className="weatherParams-item"
      >
        <div
          className="add-icon"
        />
        <div
          className="add-text"
        >
          Add
        </div>
      </div>
    </div>
  </div>
  <div
    className="chart-container"
    onScroll={[Function]}
  >
    <div
      className="chart-title"
    >
      <span
        className="blue-title"
      >
        Tide
      </span>
      <span
        className="orange-title"
      >
        Sunrise & Sunset
      </span>
    </div>
    <canvas
      className="chart-canvas"
    />
    <img
      alt="sun"
      className="img-sun"
      src="sun.svg"
    />
    <span
      className="time-text"
    >
      6:00 am
    </span>
    <div
      className="day-text"
    >
      Day 
      1
    </div>
  </div>
</div>
`;
